pages/mesosphere/dcos/services/dse/3.1.0-6.7.5/release-notes/index.md

```diff
@@ -1,17 +1,48 @@
 ---
 layout: layout.pug
 navigationTitle: Release Notes
-excerpt: Release Notes for DSE v. 3.0.0-6.7.2
+excerpt: Release Notes for DSE v. 3.1.0-6.7.5
 title: Release Notes
 menuWeight: 10
 model: /mesosphere/dcos/services/dse/data.yml
 render: mustache
 ---
 
+# Version 3.1.0-6.7.5
+
+## Updates
+- Upgraded {{ model.techShortName }} to version 6.7.5 and {{ model.techOpsName }} to version 6.7.4
+- Upgraded SDK to 0.57.1. For more information see SDK's Release Notes for: 
+  - [0.57.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.1)
+  - [0.57.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.57.0)
+  - [0.56.3](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.3)
+  - [0.56.2](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.2)
+  - [0.56.1](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.1)
+  - [0.56.0](https://github.com/mesosphere/dcos-commons/releases/tag/0.56.0)
+  - [0.55.5](https://github.com/mesosphere/dcos-commons/releases/tag/0.55.5)
+
+## New Features
+- Added support for DSS volume profiles
+- Added [custom domain](/mesosphere/dcos/services/{{ model.serviceName }}/latest/security/#forwarding-dns-and-custom-domain) support
+
+
+## Improvements
+- Remove non-HTTPS resources and references by replacing them with HTTPS ones
+- Added more directories to the PATH (resources/cassandra/bin, resources/cassandra/tools/bin)
+- Added descriptions for configuration parameters
+- Updated UI URLs to redirect to correct reference doc pages
+- Exclude {{ model.techOpsName }} keyspace from list of keyspaces managed by Repair Services
+- Accurate Scheduler Plan status in the UI
+
+## Bug Fixes
```

Comment:
Also, add an item about enabling DSEFS separately from DSE Analytics

Reply:
please re-review it Alex.